macros/nuPRISMBeamTest.mac

```diff
@@ -0,0 +1,164 @@
+# nuPRISMBeamTest WCTE macro with no visualization
+
+/run/verbose 1
+/tracking/verbose 0
+/hits/verbose 0
+#/control/execute vis.mac
+#/vis/open OGLSQt
+
+## select the geometry
+# Default config if you do nothing is currently SuperK
+# Nominal geometry for WCTE is nuPRISMBeamTest_16cShort_mPMT which is uncommented
+
+#/WCSim/WCgeom nuPRISM
+
+#Select which PMT to use
```

Comment:
should be consolidated with mPMT/photosensor settings below? or remove this section entirely since not an option for WCTE?